Packs/CommonTypes/IncidentFields/incidentfield-IP_Reputation.json

```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "ipreputation",
```

Comment:
Why do we need an IP Reputation field?

Reply:
showing the IP reputation in the layout.